path_config.go

```diff
@@ -188,9 +210,47 @@ func (b *azureSecretBackend) pathConfigWrite(ctx context.Context, req *logical.R
 		return logical.ErrorResponse(merr.Error()), nil
 	}
 
+	// set up rotation after everything is fine
+	var rotOp string
+	if config.ShouldDeregisterRotationJob() {
+		rotOp = "deregistration"
+		// Ensure de-registering only occurs on updates and if
+		// a credential has actually been registered (rotation_period or rotation_schedule is set)
+		deregisterReq := &rotation.RotationJobDeregisterRequest{
+			MountPoint: req.MountPoint,
+			ReqPath:    req.Path,
+		}
+		if prevRotateCfg {
```

Comment:
Don't think this is a blocker, but this may be inconsistent with AWS and DB currently, since they do not check for the previous config. I don't think it is wrong to do so though, just depends on what we deem to be the best

Reply:
removed